src/app/qan/profile-details/components/details-row/details-row.component.html

```diff
@@ -28,7 +28,7 @@
 </div>
 <div class="row__item row__cells">
   <span>
-    {{ isStats ? (currentMetric?.stats?.avg | humanize: pipeInfo?.perQueryStatsPipe) : '0' }}
+    {{ isStats ? (currentMetric?.stats?.avg | humanize: pipeInfo?.perQueryStatsPipe) : isSum ? perQueryStat() : '0' }}
```

Comment:
Did we decide to show '0' instead of '-' (dash) here?

Reply:
idk, there was '0', so I decided to keep it.

Reply:
for now `0` will be ok because expectation - we'll have some number there always

Reply:
let's merge then